cmd/flags.go

```diff
@@ -284,6 +284,11 @@ func metaFlags() []cli.Flag {
 			Name:  "sort-dir",
 			Usage: "sort entries within a directory by name",
 		},
+		&cli.BoolFlag{
+			Name:  "readdir-stream",
```

Comment:
For small dirs, this should be as fast as before, so this should be enabled by default, we don't need this flag